rplugin/python3/translate.py

```diff
@@ -20,6 +62,10 @@ def __init__(self, nvim):
         # if you are using different language, this may set as ""
         self.v_line_spacer = self.nvim.vars.get('translate_v_line_spacer', " ")
 
+        # NOTE: The config is auto use 'pop' for you
+        # setup display options, status, pop
+        self.display_option = DisplayType.STATUS if self.nvim.vars.get('translate_display_option') == 'status' else DisplayType.POPUP
```

Comment:
The colors maybe set as additional options `translate_bg_color` & `translate_fg_color` here.  Thanks :)

Reply:
Sounds good! I added support for `translate_display_colortype`, `translate_fg_color`, and `translate_bg_color` variables. `translate_display_colortype` is used to properly map the string colors to the nvim equivalent. It is expected to be either 8 or 16, and the program defaults to 8 if the variable has not been set.